cynic/src/coercions.rs

```diff
@@ -13,6 +13,12 @@ use crate::Id;
 pub trait CoercesTo<T> {}
 
 impl<T, TypeLock> CoercesTo<Option<TypeLock>> for Option<T> where T: CoercesTo<TypeLock> {}
+impl<T, TypeLock> CoercesTo<Option<TypeLock>> for MaybeUndefined<T> where T: CoercesTo<TypeLock> {}
+impl<T, TypeLock> CoercesTo<MaybeUndefined<TypeLock>> for Option<T> where T: CoercesTo<TypeLock> {}
+impl<T, TypeLock> CoercesTo<MaybeUndefined<TypeLock>> for MaybeUndefined<T> where
+    T: CoercesTo<TypeLock>
+{
+}
```

Comment:
Are you sure these `CorecesTo<MaybeUndefined<...` impls are needed?  I feel like they might not be....